index.js

```diff
@@ -204,11 +204,24 @@ function markdownTable(table, options) {
     rule = [];
 
     while (++index < cellCount) {
+      /* When `straighten` is false, make the rule the same size as the first row. */
+      if (settings.straighten === false) {
+        value = table[0][index];
+        if (value === null || value === undefined) {
+          value = '';
+        } else {
+          value = String(value);
+        }
+        spacing = value.length > 3 ? value.length : 3;
```

Comment:
Could you add the `3` as a constant in the root scope? Something like `var MAX_FOO = 3`, together with `spacing = value.length > MAX_FOO ? value.length : MAX_FOO;`